VSDiagnostics/VSDiagnostics/VSDiagnostics/Diagnostics/Attributes/OnPropertyChangedWithoutCallerMemberName/OnPropertyChangedWithoutCallerMemberNameCodeFix.cs

```diff
@@ -51,9 +51,9 @@ public override ImmutableArray<string> FixableDiagnosticIds
 
             editor.ReplaceNode(param, newParam);
 
-            var parentClass = methodDeclaration.Ancestors().OfType<ClassDeclarationSyntax>().FirstOrDefault();
+            var parentNode = GetParent(methodDeclaration);
```

Comment:
You can use the helper I've introduced in #568: `SyntaxNode.GetEnclosingTypeNode()`